runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/EditorWidget.tsx

```diff
@@ -12,6 +12,9 @@ const EditorWidget: FC<WidgetProps> = (props) => {
     const language: string = props.uiSchema['ui:options']?.language
         ? (props.uiSchema['ui:options']?.language as string)
         : 'typescript';
+    const isReadOnly = props.uiSchema['ui:options']?.readonly
+        ? true : false;
```

Comment:
you can parse readonly property to Boolean eg.
`const isReadOnly = !!props.uiSchema['ui:options']?.readonly`
or
`const isReadOnly = Boolean(props.uiSchema['ui:options']?.readonly)`